tests/ha/test_server_pod_failure.py

```diff
@@ -1291,15 +1185,15 @@ def test_deletes_during_server_pod_down(self):
                                   f"Bucket DELETEs failed when no server pod down {fail_del_bkt}")
         assert_utils.assert_true(len(event_del_bkt),
                                  f"No bucket DELETEs failed during server pod down {event_del_bkt}")
-        LOGGER.info("Step 7: Verified status for In-flight DELETEs while server pod %s going "
+        LOGGER.info("Step 4: Verified status for In-flight DELETEs while server pod %s going "
                     "down", pod_name)
 
         LOGGER.info("Cleaning up IAM user data")
         resp = self.ha_obj.delete_s3_acc_buckets_objects(self.s3_clean)
         assert_utils.assert_true(resp[0], resp[1])
         self.s3_clean.pop('s3_acc')
 
-        LOGGER.info("Step 8: Create IAM user with multiple buckets and run IOs when cluster has "
+        LOGGER.info("Step 5: Create IAM user with multiple buckets and run IOs when cluster has "
                     "some failures due to server pod %s going down.", pod_name)
```

Comment:
Instead of "when cluster has "
                    "some failures due to server pod %s going down."
can we say when server pod is down.

Because server pod is already down.